From 02e79a91f97f18f77706cbe7aab42eb6157be0fd Mon Sep 17 00:00:00 2001
From: Preston Petrie <preston159@users.noreply.github.com>
Date: Fri, 9 May 2025 14:31:42 -0500
Subject: [PATCH 2/6] add software

---
 Containerfile | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/Containerfile b/Containerfile
index cebab8b3..e760c97d 100644
--- a/Containerfile
+++ b/Containerfile
@@ -48,6 +48,7 @@ FROM ghcr.io/ublue-os/akmods-extra:${KERNEL_FLAVOR}-${FEDORA_VERSION}-${KERNEL_V
 
 FROM scratch AS ctx
 COPY build_files /
+COPY ../build.sh /build.sh
 
 ################
 # DESKTOP BUILDS
@@ -400,6 +401,15 @@ RUN --mount=type=cache,dst=/var/cache \
     chmod +x /usr/libexec/bazzite-yafti-launcher && \
     /ctx/cleanup
 
+# my packages
+RUN --mount=type=cache,dst=/var/cache \
+    --mount=type=cache,dst=/var/log \
+    --mount=type=bind,from=ctx,source=/,target=/ctx \
+    --mount=type=tmpfs,dst=/tmp \
+    mkdir -p /var/lib/alternatives && \
+    /ctx/build.sh && \
+    /ctx/cleanup
+
 # Configure KDE & GNOME
 RUN --mount=type=cache,dst=/var/cache \
     --mount=type=cache,dst=/var/log \
@@ -435,7 +445,7 @@ RUN --mount=type=cache,dst=/var/cache \
             plasma-welcome-fedora \
             plasma-browser-integration \
             kcharselect \
-            konsole && \
+            && \
         sed -i '/<entry name="launchers" type="StringList">/,/<\/entry>/ s/<default>[^<]*<\/default>/<default>preferred:\/\/browser,applications:steam.desktop,applications:net.lutris.Lutris.desktop,applications:org.gnome.Ptyxis.desktop,applications:org.kde.discover.desktop,preferred:\/\/filemanager<\/default>/' /usr/share/plasma/plasmoids/org.kde.plasma.taskmanager/contents/config/main.xml && \
         sed -i '/<entry name="favorites" type="StringList">/,/<\/entry>/ s/<default>[^<]*<\/default>/<default>preferred:\/\/browser,steam.desktop,net.lutris.Lutris.desktop,systemsettings.desktop,org.kde.dolphin.desktop,org.kde.kate.desktop,org.gnome.Ptyxis.desktop,org.kde.discover.desktop,system-update.desktop<\/default>/' /usr/share/plasma/plasmoids/org.kde.plasma.kickoff/contents/config/main.xml && \
         sed -i 's@\[Desktop Action new-window\]@\[Desktop Action new-window\]\nX-KDE-Shortcuts=Ctrl+Alt+T@g' /usr/share/applications/org.gnome.Ptyxis.desktop && \
-- 
2.49.0

